pkg/cmd/cli/cmd/examples.go

```diff
@@ -0,0 +1,81 @@
+package cmd
+
+import (
+	"fmt"
+	"io"
+
+	kapi "github.com/GoogleCloudPlatform/kubernetes/pkg/api"
+	"github.com/GoogleCloudPlatform/kubernetes/pkg/kubectl"
+	kcmdutil "github.com/GoogleCloudPlatform/kubernetes/pkg/kubectl/cmd/util"
+	"github.com/GoogleCloudPlatform/kubernetes/pkg/runtime"
+	"github.com/spf13/cobra"
+)
+
+const examplesLong = `Displays one big list of examples for most commands available in the OpenShift client tools.
```

Comment:
I didn't get an answer for why I want this command as a user if I already have the individual commands.

> On Jun 25, 2015, at 12:17 PM, Jordan Liggitt <notifications@github.com> wrote:
> 
> In pkg/cmd/cli/cmd/examples.go:
> 
> > @@ -0,0 +1,81 @@
> > +package cmd
> > +
> > +import (
> > +	"fmt"
> > +	"io"
> > +
> > +	kapi "github.com/GoogleCloudPlatform/kubernetes/pkg/api"
> > +	"github.com/GoogleCloudPlatform/kubernetes/pkg/kubectl"
> > +	kcmdutil "github.com/GoogleCloudPlatform/kubernetes/pkg/kubectl/cmd/util"
> > +	"github.com/GoogleCloudPlatform/kubernetes/pkg/runtime"
> > +	"github.com/spf13/cobra"
> > +)
> > +
> > +const examplesLong = `Displays one big list of examples for most commands available in the OpenShift client tools.
> "Displays examples for most commands in the OpenShift client tools."
> 
> —
> Reply to this email directly or view it on GitHub.
>